code/modules/species/station/lizard.dm

```diff
@@ -72,11 +72,11 @@
 	base_auras = list(
 		/obj/aura/regenerating/human/unathi
 		)
-
+/*Urist Edit: Removes Unathi Regen.
```

Comment:
No it does not. You've just _disabled the toggle_. 

Meaning that if the regen started out enabled, there's no way to **dis**able it without having staff do it manually.

Reply:
Crap okay I think I figured it out.